src/_data/var.yml

```diff
@@ -21,14 +21,21 @@ mbi: Magento Business Intelligence
 # Cloud product name variables
 
 ece: Magento Commerce Cloud
+ece-release-date: June 25, 2020
```

Comment:
I added the variable for release date because it's used in 4 different files, and I don't want to have to go back and change the value in all four topics. Using this variable also prevents merge conflicts when working on different package branches.

Reply:
I wasn't worried about release date specifically, but the list of variables (in general) is long.